tools/fuchsia/fuchsia_libs.gni

```diff
@@ -40,27 +58,35 @@ common_libs = [
   },
   {
     name = "ld.so.1"
-    path = rebase_path("$sysroot_dist_lib")
+    path = rebase_path("$sysroot_dist_lib/$ld_so_path")
   },
+]
 
-  # Note, we use the md5 hashes here because of gn limitations of json parsing.
-  # This is a hack, and we can migrate to a better way soon.
+# Note, for clang libs we use the md5 hashes here because of gn limitations of
+# json parsing.
+# This is a hack, and we can migrate to a better way soon.
+common_libs += [
```

Comment:
Why do we need to `+=` these libraries here instead of keeping them as part of the initial definition of `common_libs` above?

Reply:
They are supplied by the clang toolchain, as opposed to the other libraries which are supplied by the fuchsia SDK.  So I wanted to break them out